### DISCUSSION TITLE
Alternative to decimals for specifying technical unit precision (e.g. Watthours)

### DISCUSSION THREAD

**Discussion by marcelgsteiger** [grafana/grafana#38493] [Dashboards, Panels & Visualizations]
Hi all
I am currently building a smartmeter for a photovoltaic installation using grafana 8, PostgreSQL 11 and Zabbix 4 on a raspberry pi 4. Now I have a suggestion for displaying energy values.
My system measures in 1/10 watthours. To suppress decimals, I can specify zero decimal places.
Now when the value is above 1000, the system automatically converts to kilowatthours. This way I get the reading 999 Wh , then with 1 Wh more, I just get 1 kWh. The reading stays here until I reach 2 kWh. The same happens when I go from kWh to MWh.
This makes the decimal selector pretty unusable in this case.
I could imagine two solutions:
1) fixing the unit, e.g. on kWh, disabling the scaling automatism to MWh / GWh (Zabbix goes this way, for instance)
2) a way to specify the minimum number of sigificant digits instead of the number of decimals, e.g. 3. This way, I could get
999 W
1.00 kW
1.01 kW...
10.0 kW
10.1 kW
etc.
This would make several of my graphics, e.g. pie charts more readable and informative.

I am pretty new to grafana, so please forgive me if this can already be achieved in some way and I just didn't find out how. I would not be surprised...

Best regards
Marcel

**Comment by leeoniya** [comment]
somewhat related: https://github.com/grafana/grafana/issues/35838

**Reply by marcelgsteiger** [reply]
you are right - this seems to be similar to https://github.com/grafana/grafana/issues/8621 .

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
